scala/pekkohttp/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.9.9
```

Comment:
this is driven by container provided version, could you remove this file ?

Reply:
Done